alcs-frontend/src/app/features/admin/noi-subtype/noi-subtype.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="container">
+  <div class="split">
+    <h3>NOI Subtypes</h3>
+    <button mat-flat-button color="primary" (click)="onCreate()">Create</button>
+  </div>
+
+  <table mat-table [dataSource]="subtypes">
+    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+    <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+
+    <ng-container matColumnDef="number">
+      <th mat-header-cell *matHeaderCellDef>Number</th>
+      <td mat-cell *matCellDef="let row">{{ row.number }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="label">
+      <th mat-header-cell *matHeaderCellDef>Label</th>
+      <td mat-cell *matCellDef="let row">{{ row.label }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="description">
+      <th mat-header-cell *matHeaderCellDef>Description</th>
+      <td mat-cell *matCellDef="let row">{{ row.description }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="code">
+      <th mat-header-cell *matHeaderCellDef>Code</th>
+      <td mat-cell *matCellDef="let row">{{ row.code }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="isActive">
+      <th mat-header-cell *matHeaderCellDef>Status</th>
+      <td mat-cell *matCellDef="let row">
+        <div class="left" *ngIf="row.isActive">
+          <mat-icon color="primary">check_circle_outline</mat-icon>
+          Active
+        </div>
+        <div class="left" *ngIf="!row.isActive">
+          <mat-icon color="warn">not_interested</mat-icon>
+          Inactive
+        </div>
+      </td>
+    </ng-container>
+
+    <ng-container matColumnDef="actions">
+      <th mat-header-cell *matHeaderCellDef>Actions</th>
+      <td mat-cell *matCellDef="let row">
+        <button *ngIf="row.code !== 'MODI'" class="edit-btn" mat-flat-button (click)="onEdit(row)">
```

Comment:
MODI? is it from modification or it is a subtype type?

Reply:
Removed